File: 3430-count-days-without-meetings/3430-count-days-without-meetings.java
class Solution {
    public int countDays(int days, int[][] meetings) {
        Arrays.sort(meetings,(a,b)->Integer.compare(a[0],b[0]));
        int n=meetings.length;
        int st=0;
        int ans=0;
        for(int i=0;i<n;i++){
            if(meetings[i][0]>st){
                ans+=(meetings[i][0]-st-1);
                st=meetings[i][1];
            }else st=Math.max(st,meetings[i][1]);
        }return ans+days-st;
    }
}

File: 3430-count-days-without-meetings/3430-count-days-without-meetings.py
class Solution:
    def countDays(self, days: int, meetings: List[List[int]]) -> int:
        
        m = sorted(meetings)

        cnt = 0
        s = 0

        for i in range(len(meetings)):
            if m[i][0]>s:
                cnt += m[i][0] - s - 1

            s = max(s, m[i][1])

        return cnt+days-s